intel_models/ssd300-fp32-onnx-0001/description/ssd300-fp32-onnx-0001.md

```diff
@@ -0,0 +1,47 @@
+# ssd300-fp32-onnx-0001
```

Comment:
Add `Legal information` part (see other model's description for example).

Reply:
Done.